lisa/runners/legacy_runner.py

```diff
@@ -389,7 +389,8 @@ class LogParser(InitializableMixin):
     # ')
     CASE_COMPLETED = re.compile(
         r"SQLQuery\:  INSERT INTO LISATestTelemetry \(.*\) "
-        r"VALUES \('.*?','.*?','(?P<location>.*?)','.*?','.*?','(?P<name>.*?)',"
+        r"VALUES \('.*?','(?P<platform>.*?)','(?P<location>.*?)',"
```

Comment:
Make sure it's exact the same as name in v3, including letter case. If not, we should convert it in code.